accounts/abi/bind/backends/simulated.go

```diff
@@ -63,11 +63,10 @@ type SimulatedBackend struct {
 	database   ethdb.Database   // In memory database to store our testing data
 	blockchain *core.BlockChain // Ethereum blockchain to handle the consensus
 
-	mu                sync.Mutex
-	stuckTransactions types.Transactions // holds onto all txes that don't go into the pending block due to low gas
```

Comment:
this was the reason to have the fork. are we deciding to remove this and any tests that used it in pax?

Reply:
will add this back in